Marvelys-Torres/tarea-4/scripts.js

```diff
@@ -0,0 +1,151 @@
+let opera;
+let operb;
+let operation;
+let res;
+
+let result = document.getElementById("result");
+const sum = document.getElementById("sum");
+const resta = document.getElementById("resta");
+const mul = document.getElementById("mul");
+const divi = document.getElementById("divi");
+
+const equal = document.getElementById("equal");
+const dot = document.getElementById("dot");
+const reset = document.getElementById("reset");
+
+const zero = document.getElementById("zero");
+const one = document.getElementById("one");
+const two = document.getElementById("two");
+const three = document.getElementById("three");
+const four = document.getElementById("four");
+const five = document.getElementById("five");
+const six = document.getElementById("six");
+const seven = document.getElementById("seven");
+const eight = document.getElementById("eight");
+const nine = document.getElementById("nine");
+
+function cleanDisplay() {
+  if (res) {
+    res = undefined;
+    result.textContent = "";
+  }
+}
+
+dot.onclick = function () {
+  cleanDisplay();
+  if (result.textContent.includes(".")) return;
+  result.textContent = result.textContent + ".";
+};
+zero.onclick = function () {
+  cleanDisplay();
+  if (result.textContent.length > 10) return;
+  result.textContent = result.textContent + "0";
+};
+one.onclick = function () {
+  cleanDisplay();
+  if (result.textContent.length > 10) return;
+  result.textContent = result.textContent + "1";
```

Comment:
@MarvelysTorres , look to me like you can do only one function for all the onclick listener, if you pay attention the only thing that is differnt is this line

```suggestion
  result.textContent = result.textContent + event.target.value;
```